playbooks/vars/defaults.yaml

```diff
@@ -69,3 +69,8 @@ neutron_flat_networks: "external,hostonly"
 tripleo_repos_repos:
   - current-tripleo-dev
   - ceph
+
+ceph_enabled: false
```

Comment:
This is the kind of config complexity I like 😁

Any idea how much time/resource it adds to the installation? IOW, any reason not to make the default true?

Reply:
it's adding a good 15min and it's a loop device, so perfs aren't great, I wouldn't make it the default for now.